ui/components/app/account-menu/account-menu.component.js

```diff
@@ -423,11 +425,23 @@ export default class AccountMenu extends Component {
           icon={
             <img
               className="account-menu__item-icon"
-              src="images/settings.svg"
+              src="./images/alphaCarbon/settings.svg"
```

Comment:
- I think the path should be like this
```suggestion
              src="images/alphaCarbon/settings.svg"
```

Reply:
fixed in e19c10d3b8cc3d6dc37987e65e8a71c9911fe953

Reply:
fixed in e19c10d3b8cc3d6dc37987e65e8a71c9911fe953